Cargo.toml

```diff
@@ -4,4 +4,4 @@ members = [
     "matchbox_server",
     "examples/*",
 ]
-resolver = "2" # Necessary or WGPU has issues
+resolver = "2" # Important! Bevy/WGPU needs this!
```

Comment:
This likely is a thing that will likely be in every rust workspace going forward, unless you want to use the legacy resolver :/ https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html#details

Reply:
I noticed as well. I thought at first it wasn't needed since 2021 uses resolver="2" by default. Interesting note.

Reply:
yeah, but I don't think the workspace actually has an edition, only the crates do.